internal/cmd/serve.go

```diff
@@ -51,6 +53,7 @@ func serveHandler(cmd *cobra.Command, args []string) {
 		ServerAddress: address,
 		ServerPort:    port,
 		RootPath:      rootPath,
+		Log:  				 log,
```

Comment:
Is this unproperly formatted or it's Github's diff view being funny? (It's happening on other lines as well)

Reply:
mmm.. No probs in VS Code and nothing with `go fmt` but, let me double check,

Reply:
It's all bonkers, let me reformat this properly. Dunno what happened.